src/javaRosa/itextWidget.js

```diff
@@ -0,0 +1,478 @@
+define([
+    'underscore',
+    'jquery',
+    'tpl!vellum/templates/auto_box',
+    'tpl!vellum/templates/markdown_help',
+    'vellum/javaRosa/util',
+    'vellum/widgets',
+    'vellum/util',
+    'vellum/atwho',
+    'vellum/core'
+], function (
+    _,
+    $,
+    auto_box,
+    markdown_help,
+    jr_util,
```

Comment:
also noticed the same in javaRosa.js, but the diff is not available for comment on github